src/test/kotlin/calculator/OperatorTest.kt

```diff
@@ -0,0 +1,35 @@
+package calculator
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class OperatorTest {
+    @Test()
+    fun `findBy 메서드는 전달받은 symbol과 일치하는 연산자 enum을 반환한다`() {
```

Comment:
```suggestion
    fun `전달받은 symbol과 일치하는 연산자를 반환한다`() {
```
주어를 빼도 이해하는데 문제 없어 보여요 (이하 관련 내용 동일)